src/system_program.rs

```diff
@@ -94,6 +98,14 @@ impl SystemProgram {
                     accounts[0].tokens -= tokens;
                     accounts[1].tokens += tokens;
                 }
+                SystemProgram::Spawn => {
+                    if !accounts[0].executable || accounts[0].loader_program_id != Pubkey::default()
+                    {
+                        Err(Error::AssignOfUnownedAccount)?;
```

Comment:
AssignOfUnownedAccount: emitting an Assign error from Spawn seems is a little confusing to me